### PR TITLE
Translating BP-DR-6

### PR DESCRIPTION
**Pull request by gpressutto5**

### CODE REVIEW

**Comment by andercampanha** [comment]
This changes may have broken other error messages, like absent CVV error.

File: woocommerce-gateway-ebanx/gateways/class-wc-ebanx-gateway.php [hunk 1]
```diff
@@ -783,6 +770,35 @@ public function process_payment($order_id)
 		}
 	}
 
+	private static function get_error_message($exception, $country)
```
Comment (cezarlz) [inline woocommerce-gateway-ebanx/gateways/class-wc-ebanx-gateway.php]: Are you missing the method comments?

File: woocommerce-gateway-ebanx/gateways/class-wc-ebanx-gateway.php [hunk 2]
```diff
@@ -783,6 +770,35 @@ public function process_payment($order_id)
...
+			'co' => 'es',
+			'br' => 'pt-br',
+		);
+		$language = $languages[$country];
+
+		$errors = WC_EBANX_Errors::get_errors();
+
+		if ($code === 'BP-DR-6' && $language === 'es') {
+			$error_info = array();
+			preg_match('/Amount must be greater than (\w{3}) (.+)/',
```
Comment (cezarlz) [inline woocommerce-gateway-ebanx/gateways/class-wc-ebanx-gateway.php]: I think we must wrap it into the method `__`.

Comment (gpressutto5) [inline woocommerce-gateway-ebanx/gateways/class-wc-ebanx-gateway.php]: Why would I want to internationalize a regex?

**Pull request merged**